client/src/index.js

```diff
@@ -6,12 +6,6 @@ import 'bootstrap/dist/css/bootstrap.min.css'; // for bootstrap css library
 
 import App from './components/App';
 
-/**
- *  For react-bootstrap
- */
-import 'bootstrap/dist/css/bootstrap.css';
-import 'bootstrap/dist/css/bootstrap-grid.css';
-import 'bootstrap/dist/css/bootstrap-reboot.css';
```

Comment:
Is there a reason we're removing this?

Reply:
See this comment above https://github.com/Researchify/Researchify/pull/150#discussion_r686726239